docs/v1/user-story.md

```diff
@@ -0,0 +1,124 @@
+# A Year In The Life Of A Kubernetes Service Developer
+
+## 0. Today
+I'ma developer in an enterprise environment.  For some fun in my spare time, I’m building a node.js chat application for me 
+and a couple of coworkers to goof around with.  It’s really simple.  It runs on one container.  To get going, I simply 
+download a node.js container onto minikube on my laptop and started coding.  I want some co-workers to be able to find the 
```

Comment:
I also sort of feel like the progression of this story might be easier to digest if the narrator begins with consuming a service.  To me consuming a service is the most primitive use-case and running your own broker is a more advanced one.  So, maybe the flow can be:

1.  Narrator makes chat app in local kube
2.  Narrator consumes rabbit MQ service
3.  Narrator needs to move off of local kube, registers in service catalog...